dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -449,16 +451,18 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws Servl
 				    GregorianCalendar expiration = new GregorianCalendar();
 					expiration.add(java.util.Calendar.MONTH, -1);
 					resp.setHeader("Expires", DownloadUtil.httpDate.get().format(expiration.getTime()));
-					resp.setHeader("Cache-Control", "max-age=-1");
+					if (!resp.containsHeader(HttpHeaders.CACHE_CONTROL)) {
+						resp.setHeader(HttpHeaders.CACHE_CONTROL, "max-age=-1");
+					}
 				}
 			}
 
 			String rangeHeader = req.getHeader("range");
 			if(UtilMethods.isSet(rangeHeader)){
 
-				try {
+				try (FileInputStream fileInputStream = new FileInputStream(data.getDataFile())) {

```

Comment:
Do not we used a diff approach for FileInputStream (from NIO) to about issues with finalize